src/constants/index.tsx

```diff
@@ -247,58 +245,87 @@ export const PINNED_PAIRS: {
 
 export const ARBITRUM_ONE_PROVIDER = new providers.JsonRpcProvider('https://arb1.arbitrum.io/rpc')
 
+export enum ConnectorType {
+  // INJECTED = 'INJECTED',
```

Comment:
do we need this comment?

Reply:
Removed [a5d92a4](https://github.com/levelkdev/swapr-dapp/pull/1330/commits/a5d92a4255550303e4b0586fb90a05b0f4344d52)